tensorflow/compiler/tf2tensorrt/BUILD

```diff
@@ -370,7 +384,6 @@ cc_library(
     name = "trt_op_libs",
     deps = [
         ":get_calibration_data_op_op_lib",
-        ":trt_convert_api",
```

Comment:
Can you pass tensorflow/compiler/tf2tensorrt:convert_qdq_test_gpu?
I got this error: third_party/tensorflow/compiler/tf2tensorrt/convert/ops/quantization_ops_test.cc:33:10: fatal error: 'third_party/tensorflow/compiler/tf2tensorrt/trt_convert_api.h' file not found
#include "third_party/tensorflow/compiler/tf2tensorrt/trt_convert_api.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~